core/src/main/java/discord4j/core/object/entity/GuildChannel.java

```diff
@@ -47,6 +48,16 @@
      * @return The permission overwrites for this channel.
      */
     Set<PermissionOverwrite> getPermissionOverwrites();
+    
+    /**
+     * Utility to get all effective permissions for a user in this channel.
```

Comment:
I think this could be reworded to "Gets the given member's effective permissions in this channel."